lib/util/http/github.spec.ts

```diff
@@ -786,4 +786,38 @@ describe('util/http/github', () => {
       ).rejects.toThrow(EXTERNAL_HOST_ERROR);
     });
   });
+
+  describe('getRawFile()', () => {
+    it('add header and return', async () => {
+      httpMock
+        .scope(githubApiHost)
+        .get('/foo/bar/contents/lore/ipsum.txt')
+        .matchHeader(
+          'accept',
+          'application/vnd.github.raw+json, application/vnd.github.v3+json',
+        )
+        .reply(200, 'foo');
+      await expect(
+        githubApi.getRawFile(
+          `${githubApiHost}/foo/bar/contents/lore/ipsum.txt`,
+        ),
+      ).resolves.toMatchObject({
+        body: 'foo',
+      });
+    });
+
+    it('support relative path', async () => {
+      httpMock
+        .scope(githubApiHost)
+        .get('/foo/bar/contents/lore/ipsum.txt')
+        .reply(200, 'foo');
+      await expect(
+        githubApi.getRawFile(
+          `${githubApiHost}/foo/bar/contents/foo/../lore/ipsum.txt`,
```

Comment:
```suggestion
          `foo/bar/contents/foo/../lore/ipsum.txt`,
```
I mean this

Reply:
No that isn't supported

Reply:
Actually, it has worked already. 

I have added additionally logic to allow simply querying by providing the path to a file and the repository as option.